src/nodes/basic.ts

```diff
@@ -11,6 +11,10 @@ export const docParagraph: NodeSpec = {
   content: 'paragraph',
 };
 
+export const commentDoc: NodeSpec = {
+  content: `(paragraph | ${NodeGroups.block})+`, // browsers will completely collapse the node when it's empty `+` is necessary
```

Comment:
This can just be `` `${NodeGroups.block}+` ``, as paragraph is a block.

Should there be headings in here do you think?